From 2057215f36d2abd564ec547a8ec0a2edde1d22c2 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Sat, 24 Feb 2018 15:37:44 +0800
Subject: [PATCH] Revert "arm64: rockchip_defconfig: remove
 CONFIG_ANDROID_LOW_MEMORY_KILLER"

This reverts commit 7adc50bef2c77d9caad22ae52beed521e885df18.

Failed to pass vintf compatibility test.

Change-Id: Ia8f3d3be1b65b194cea68dc495a2718f7b2efbfc
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 01541e6f1386..28c31d3914e8 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -699,6 +699,7 @@ CONFIG_STAGING=y
 CONFIG_INV_MPU_IIO=y
 CONFIG_ASHMEM=y
 CONFIG_ANDROID_TIMED_GPIO=y
+CONFIG_ANDROID_LOW_MEMORY_KILLER=y
 CONFIG_SYNC=y
 CONFIG_SW_SYNC=y
 CONFIG_SW_SYNC_USER=y
-- 
2.35.3

